pkg/cluster/storageSuffix.go

```diff
@@ -0,0 +1,36 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func openShiftClusterDocumentMutatorFn(doc *api.OpenShiftClusterDocument) error {
```

Comment:
nit: can we rename this to `mutateStorageSuffix` so it is inline with the other 'mutate' function in this PR?

Reply:
Good idea. Done. 😄